### PR TITLE
refine dataloder doc

### PR DESCRIPTION
**Pull request by heavengate**
**refine dataloder doc**
cn doc for https://github.com/PaddlePaddle/Paddle/pull/23107

- DataLoader
dataaloder1
dataloader2
dataloader3
dataloader4
dataloader5

- Dataset
dataset

- BatchSampler
batch_sampler1
batch_sampler2

### CODE REVIEW

File: doc/fluid/api_cn/io_cn/BatchSampler_cn.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+``__iter__`` : 迭代返回mini-batch索引列表。
+
+``__len__`` : 每个epoch中的mini-batch个数。
+
+参数:
+    - **dataset** (Dataset) - ``fluid.io.Dataset`` 实例或者实现了 ``__len__`` 接口的python对象，用于生成 ``dataset`` 长度范围的索引。默认值为None。
+    - **indices** (list|tuple) - 用于迭代的下标，``dataset`` 的替代参数， ``dataset`` 和 ``indices`` 必须设置其中之一。默认值为None。
+    - **shuffle** (bool) - 迭代返回索引之前是否对索引打乱顺序。默认值为False。
+    - **batch_size** (int) - 每mini-batch中的索引下标个数。默认值为1。
+    - **drop_last** (int) - 是否丢弃因数据集样本数不能被 ``batch_size`` 整除而产生的最后一个不完成的mini-batch索引。默认值为False。
```
Comment (Boyan-Liu) [inline doc/fluid/api_cn/io_cn/BatchSampler_cn.rst]: 不完成->不完整 是否会更容易理解？

File: doc/fluid/api_cn/io_cn/DataLoader_cn.rst [hunk 2]
```diff
@@ -3,7 +3,139 @@
...
+    - **places** (list(Place)|tuple(Place)) - 数据需要放置到的Place列表。在静态图和动态图模式中，此参数均必须设置。在动态图模式中，此参数列表长度必须是1。默认值为None。
+    - **return_list** (bool) - 每个设备上的数据是否以list形式返回。若return_list = False，每个设备上的返回数据均是str -> LoDTensor的映射表，其中映射表的key是每个输入变量的名称。若return_list = True，则每个设备上的返回数据均是list(LoDTensor)。在动态图模式下，此参数必须为True。默认值为False。
+    - **batch_sampler** (BatchSampler) - ``fluid.io.BatchSampler`` 或其子类的实例，DataLoader通过 ``batch_sampler`` 产生的mini-batch索引列表来 ``dataset`` 中索引样本并组成mini-batch。默认值为None。
+    - **batch_size** (int) - 每mini-batch中样本个数，为 ``batch_sampler`` 的替代参数，若 ``batch_sampler`` 未设置，会根据 ``batch_size`` ``shuffle`` ``drop_last`` 创建一个 ``fluid.io.BatchSampler`` 。默认值为1。
+    - **shuffle** (bool) - 生成mini-batch索引列表时是否对索引打乱顺序，为 ``batch_sampler`` 的替代参数，若 ``batch_sampler`` 未设置，会根据 ``batch_size`` ``shuffle`` ``drop_last`` 创建一个 ``fluid.io.BatchSampler`` 。默认值为False。
+    - **drop_last** (bool) - 是否丢弃因数据集样本数不能被 ``batch_size`` 整除而产生的最后一个不完成的mini-batch，为 ``batch_sampler`` 的替代参数，若 ``batch_sampler`` 未设置，会根据 ``batch_size`` ``shuffle`` ``drop_last`` 创建一个 ``fluid.io.BatchSampler`` 。默认值为False。
```
Comment (Boyan-Liu) [inline doc/fluid/api_cn/io_cn/DataLoader_cn.rst]: 不完成，同上

File: doc/fluid/api_cn/io_cn/BatchSampler_cn.rst (hunk 1 above)
Comment (heavengate) [reply doc/fluid/api_cn/io_cn/BatchSampler_cn.rst]: Done, thanks!

File: doc/fluid/api_cn/io_cn/DataLoader_cn.rst (hunk 2 above)
Comment (heavengate) [reply doc/fluid/api_cn/io_cn/DataLoader_cn.rst]: Done, thanks!

**Review by Boyan-Liu — APPROVED** [review]

**Pull request closed without merging**